package/clojure/parsetab.py

```diff
@@ -7,7 +7,7 @@
+_lr_action_items = {'LBRACE':([0,1,2,3,4,5,6,7,8,9,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,],[1,1,1,-8,-6,-7,1,-9,-10,-1,-3,-2,1,-5,-11,-18,1,-4,-16,1,-14,1,-12,-17,-15,-13,]),'READMACRO':([0,1,2,3,4,5,6,7,8,9,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,],[2,2,2,-8,-6,-7,2,-9,-10,-1,-3,-2,2,-5,-11,-18,2,-4,-16,2,-14,2,-12,-17,-15,-13,]),'LBRACKET':([0,1,2,3,4,5,6,7,8,9,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,],[6,6,6,-8,-6,-7,6,-9,-10,-1,-3,-2,6,-5,-11,-18,6,-4,-16,6,-14,6,-12,-17,-15,-13,]),'$end':([3,4,5,7,8,9,10,11,12,14,16,18,19,21,24,25,26,],[-8,-6,-7,-9,-10,-1,0,-3,-2,-5,-18,-4,-16,-14,-17,-15,-13,]),'INTEGER':([0,1,2,3,4,5,6,7,8,9,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,],[5,5,5,-8,-6,-7,5,-9,-10,-1,-3,-2,5,-5,-11,-18,5,-4,-16,5,-14,5,-12,-17,-15,-13,]),'FLOAT':([0,1,2,3,4,5,6,7,8,9,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,],[4,4,4,-8,-6,-7,4,-9,-10,-1,-3,-2,4,-5,-11,-18,4,-4,-16,4,-14,4,-12,-17,-15,-13,]),'NIL':([0,1,2,3,4,5,6,7,8,9,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,],[9,9,9,-8,-6,-7,9,-9,-10,-1,-3,-2,9,-5,-11,-18,9,-4,-16,9,-14,9,-12,-17,-15,-13,]),'RPAREN':([3,4,5,7,8,9,11,12,13,14,15,16,18,19,21,22,23,24,25,26,],[-8,-6,-7,-9,-10,-1,-3,-2,21,-5,-11,-18,-4,-16,-14,26,-12,-17,-15,-13,]),'QUOTED':([0,1,2,3,4,5,6,7,8,9,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,],[11,11,11,-8,-6,-7,11,-9,-10,-1,-3,-2,11,-5,-11,-18,11,-4,-16,11,-14,11,-12,-17,-15,-13,]),'RBRACE':([1,3,4,5,7,8,9,11,12,14,15,16,17,18,19,21,23,24,25,26,],[16,-8,-6,-7,-9,-10,-1,-3,-2,-5,-11,-18,24,-4,-16,-14,-12,-17,-15,-13,]),'ATOM':([0,1,2,3,4,5,6,7,8,9,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,],[12,12,12,-8,-6,-7,12,-9,-10,-1,-3,-2,12,-5,-11,-18,12,-4,-16,12,-14,12,-12,-17,-15,-13,]),'LPAREN':([0,1,2,3,4,5,6,7,8,9,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,],[13,13,13,-8,-6,-7,13,-9,-10,-1,-3,-2,13,-5,-11,-18,13,-4,-16,13,-14,13,-12,-17,-15,-13,]),'KEYWORD':([0,1,2,3,4,5,6,7,8,9,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,],[14,14,14,-8,-6,-7,14,-9,-10,-1,-3,-2,14,-5,-11,-18,14,-4,-16,14,-14,14,-12,-17,-15,-13,]),'RBRACKET':([3,4,5,6,7,8,9,11,12,14,15,16,18,19,20,21,23,24,25,26,],[-8,-6,-7,19,-9,-10,-1,-3,-2,-5,-11,-18,-4,-16,25,-14,-12,-17,-15,-13,]),}
```

Comment:
Generated by ply, not sure when or how it is generated and why it changes all the time